tests/scripts/k8s-vagrant-multi-node.sh

```diff
@@ -13,7 +13,7 @@ function init() {
         # checkout latest tag of the repo initially after clone
         git -C "${REPO_DIR}" checkout "$(git describe --tags `git rev-list --tags --max-count=1`)"
     else
-        git -C "${REPO_DIR}" pull || { echo "git pull failed with exit code $?. continuing as the repo is already there ..." }
```

Comment:
Just to note, the "actual" fix for the syntax error here would be to add a semicolon after the echo argument.

Reply:
Thanks @galexrt . Updated the file with the missing semicolon after the echo argument.